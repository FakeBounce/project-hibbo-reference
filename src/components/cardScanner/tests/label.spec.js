import React from 'react';
import renderer from 'react-test-renderer';

import Label from '../components/Label';

describe('Label', () => {
  const component = renderer.create(
    <Label
      label="Some label"
    />,
  ).toJSON();

  it('should render the component', () => {
    expect(component).toMatchSnapshot();
  });
});
