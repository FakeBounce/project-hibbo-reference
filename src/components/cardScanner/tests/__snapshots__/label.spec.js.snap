// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render the component 1`] = `
<View
  style={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={undefined}
  >
    Some label
  </Text>
</View>
`;
